rocky/katalogus/views/mixins.py

```diff
@@ -60,6 +60,22 @@ def is_required_field(self, field: str) -> bool:
         return self.plugin_schema and field in self.plugin_schema.get("required", [])
 
 
+class NormalizerMixin(OctopoesView):
+    """
+    When a user wants to scan one or multiple OOI's,
+    this mixin provides the methods to construct the boefjes for the OOI's and run them.
```

Comment:
The comment still talks about boefjes, not normalizers

Reply:
@underdarknl Apparently I forgot to save the file after I changed this, so it didn't get committed and pushed... It's pushed now though.